.github/workflows/tutorials.yml

```diff
@@ -0,0 +1,95 @@
+name: Tutorials
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+  # cron for every two weeks
+  schedule:
+    - cron: "0 0 * * 1"
+
+jobs:
+  hello-world-contract-unix:
+    name: Hello World Contract Tutorial Unix
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install Node.js (LTS)
+        uses: actions/setup-node@v3
+        with:
+          node-version: "lts/*"
+
+      - name: Install yarn
+        run: npm install -g yarn
+
+      - name: Install patch-package globally
+        run: npm install -g patch-package
```

Comment:
What is this used for?

Reply:
Good question. It's a global dependency introduced with the latest version of the cli. Perhaps this is a bug actually on the new cli version

Reply:
Actually the CI fails without it: https://github.com/matter-labs/tutorials/actions/runs/6312422238/job/17138440028

Reply:
Created this issue: https://github.com/matter-labs/zksync-cli/issues/57

Reply:
Given patch-package applies patches to deps on the fly perhaps our dependency tree in zksync-cli needs to be reviewed. I quick check would be to install locally in CI rather then globally to see if that works or not. If not, then its likely peer deps issue I think. 

We should not make use of this package tho, as its purpose is to fix broken node_modules. We should fix our deps if thats the case.